base/setup/usetup/lang/it-IT.h

```diff
@@ -146,19 +146,19 @@ static MUI_ENTRY itITIntroPageEntries[] =
     {
         6,
         11,
-        "ReactOS � nello stadio di sviluppo Alpha, significa che non tutte le funzioni",
+        "ReactOS \x8A nello stadio di sviluppo Alpha, significa che non tutte le funzioni",
```

Comment:
Is that supposed to be encoded in UTF-8? If so, this commit has to be reverted because USETUP (text-mode setup) breaks special characters with UTF-8.